### DISCUSSION TITLE
Styling For Add to Cart and Quantity Input

### DISCUSSION THREAD

**Discussion by digidoda** [bootscore/bootscore#625] [Ideas]
As per my other thread, please could we include the line:  flex-wrap: wrap; on line 9 of the /scss/bootscore_woocommerce/_wc_product_single.scss to stop that section from breaking when using additional plugins that are injecting product options to the page.

**Comment by crftwrk** [comment]
Yes, as I mentioned in my other comment, please create a pull request. If you do not know how to do that, I can guide you.

**Comment by digidoda** [comment]
Thank you – if you could guide me on how to do that please that would be great.

Mike

[digidoda]<https://www.digidoda.com/>
Mike Abbott
T: 03333 443 031<tel:03333443031> | M: 07795 143 658<tel:07795143658>
E: ***@***.******@***.***> | W: www.digidoda.com<https://www.digidoda.com>
[signature_577087499]<https://www.facebook.com/digidoda>
[signature_1780538199]<https://www.linkedin.com/in/digimike/>
digidoda Ltd is registered in the United Kingdom No. 10744500.
The information contained in this message is intended only for the personal and confidential use of the recipient(s) named above. If the reader of this message is not the intended recipient or an agent responsible for delivering it to the recipient, you are hereby notified that you have received this document in error and that any review, dissemination, distribution, or copying of this message is strictly prohibited. If you have received this communication in error, please notify us immediately, and delete the original message.
If you would like to read our full data policy you can find it on our website: www.digidoda.com/privacy-policy/<https://www.digidoda.com/privacy-policy/>

From: Basti ***@***.***>
Date: Monday, 13 November 2023 at 16:56
To: bootscore/bootscore ***@***.***>
Cc: Mike Abbott ***@***.***>, Author ***@***.***>
Subject: Re: [bootscore/bootscore] Styling For Add to Cart and Quantity Input (Discussion #625)

Yes, as I mentioned in my other comment, please create a pull request. If you do not know how to do that, I can guide you.

**Comment by crftwrk** [comment]
- Go to the theme page https://github.com/bootscore/bootscore and press on the top right Fork button.
- Now you have a copy of the theme in your profile.
- Do changes in your copy
- GitHub will ask you if you want to contribute and create a pull request.
- Press yes
- That's it.

Please check this issue for variable and grouped products as well. We have a release tomorrow, if you are quick we'll merge it before release.
